src/plotting.jl

```diff
@@ -7,19 +7,20 @@
     principalcoord(pc, 1), principalcoord(pc,2)
 end
 
-@recipe function f(abun::AbundanceTable; topabund::Int=10, sorton::Symbol=:top)
-    in(sorton, [:top, :hclust, abun.samples...]) || error("invalid sorton option")
+@userplot AbundancePlot
```

Comment:
I thought the plot was too specific to be the standard plot for all `ComMatrix` objects, so I've instead made it a user recipe named `abundanceplot`